core/src/main/java/com/novoda/utils/Log.java

```diff
@@ -0,0 +1,136 @@
+package com.novoda.utils;
+
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+public abstract class Log {
```

Comment:
I was going to remove it, named it `Log` to make it easier the migration and forgot to rename it afterwards. Good point.